gramps/plugins/tool/removespaces.py

```diff
@@ -53,6 +53,27 @@
 WIKI_HELP_PAGE = '%s_-_Tools' % URL_MANUAL_PAGE
 WIKI_HELP_SEC = _('manual|Remove_leading_and_trailing_spaces')
 
+def validate_lat_lon(field):
```

Comment:
This function could further be simplified to:
```
def validate_lat_lon(field):
    for char in (',', '‐', '‑', '‒', '–', '—', '―'):
        if field.find(char) != -1:
            return True
    return False
```

Reply:
I'll do that